projects/1.16/assets/timeless-and-classic-guns/tac/lang/zh_cn.json

```
@@ -0,0 +1,225 @@
+{
+    "block.tac.workbench": "枪械工作台",
+    "item.tac.m1911": "M1911",
+    "item.tac.m1894": "M1894",
+    "item.tac.m1851": "M1851",
+    "item.tac.magnumbox": "马格南子弹盒",
+    "item.tac.m1928": "汤普森 M1928",
+    "item.tac.mosin": "莫辛-纳甘步枪",
+    "item.tac.ak47": "AK-47",
+    "item.tac.box_308-block": ".308 子弹盒",
+    "item.tac.box45": ".45 ACP 子弹盒",
+    "item.tac.m60": "M60 机枪",
+    "item.tac.m1917": "M1917",
+    "item.tac.glock_17": "格洛克 17",
+    "item.tac.dp28": "DP-28 机枪",
+    "item.tac.m16a1": "M16A1",
+    "item.tac.mk18": "MK 18",
+    "item.tac.sti2011": "塔兰战术 STI 2011",
+    "item.tac.ak74": "AK-74",
+    "item.tac.m92fs": "伯莱塔 M92FS",
+    "item.tac.nato_556_box_block": "5.56口径子弹盒",
+    "item.tac.9mm_box_block": "9mm口径子弹盒",
+    "item.tac.ar_15_p": "AR-15P",
+    "item.tac.ar_15_hellmouth": "AR-15",
+    "item.tac.micro_uzi": "英格拉姆 Mac 10",
+    "item.tac.vector45": "维克托冲锋枪",
+    "item.tac.walther_ppk": "沃尔特PPK",
+    "item.tac.mosberg590": "莫斯伯格 590",
+    "item.tac.db_long": "双管猎枪",
+    "item.tac.db_short": "截短双管猎枪",
+    "item.tac.m4": "M4A1",
+    "item.tac.m24": "M24 狙击步枪",
+    "item.tac.m1911_nether": " 地狱1911",

```

```suggestion
    "item.tac.m1911_nether": " 地狱 M1911",
```
https://en.wikipedia.org/wiki/M1911_pistol